src/main/com/aca/homework/week12/temperature/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.aca.homework.week12.temperature;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        ItemWriter<DailyTemperature> writer = new DailyTemperatureFileItemWriter();
+        Random random = new Random();
+        for (int i = 1; i < 366; i++) {
+            DailyTemperature dailyTemperature = new DailyTemperature(i, random.nextInt(30) - 15);
+            writer.write(dailyTemperature);
+        }
+
+        List<DailyTemperature> dailyTemperatures = new LinkedList<>();
+        ItemReader<DailyTemperature> reader = new DailyTemperatureFileItemReader();
+        for (int i = 1; i < 366; i++) {
```

Comment:
please create a mechanism to determine when the EOF is reached. here you have hard-coded the objects number